source/type/type.h

```diff
@@ -525,6 +525,29 @@ struct VehicleJourney: public Header, Nameable, hasVehicleProperties {
     // return the base vj corresponding to this vj, return nullptr if nothing found
     const VehicleJourney* get_corresponding_base() const;
 
+    /*
+     *
+     *
+     *  Day     1              2               3               4               5               6
+     *          -----------------------------------------------------------------------------------------------------
+     * SP_bob           8:30         8:30             8:30                       (vj)
+     * Period_bob   |-----------------------------------------------|
+     *
+     *
+     * Let's say we have a vj passes every day at 8:30 on a stop point SP_bob.
+     * Now given a Period_bob and the VJ stops at SP_bob only during this Period_bob, what's the validity pattern of the
```

Comment:
Maybe precise the Period_bob to ease understanding (say... day1 06:00 to day4 06:00).
And add 8:30 on days 4-5-6 of schema.